.github/workflows/programequity_slack.yml

```
@@ -0,0 +1,24 @@
+on:
```

We should add a friendly display name for this Actions workflow:

```suggestion
name: Notify mentioned users via Slack

on:
```